docs/nx-cloud/troubleshooting/explain-with-ai.md

```diff
@@ -0,0 +1,78 @@
+# Explain with AI (beta)
+
+"Explain with AI" in Nx Cloud leverages artificial intelligence to provide detailed explanations and insights for failed task outputs.
+This feature helps developers understand complex errors quickly and offers suggestions for improvements, making the debugging process more efficient.
+
+With the power of Nx Cloud, all explanations will be provided with additional context from Nx targets and metadata. This ensures that the AI responses have the correct context related to your failed task. This means that if you're trying to debug failed Gradle tasks, the AI will know to help you specifically with Gradle.
+
+## What You Need to Enable This Feature
+
+To use the "Explain with AI" feature, you need to [enable AI features for your organization](/ci/features/nx-cloud-ai).
+
+- For enterprise on-prem customers, this feature is not enabled by default. You need to explicitly [enable AI features through environment flags](#how-to-enable-ai-features-for-enterprise-onprem-installations).
+- For public cloud users, the AI feature is enabled automatically, [except for those in the EU due to regional restrictions](#regional-availability).
+- This feature is not available for hobby (free) plans on Nx Cloud.
+
+## How to Use
+
+{% callout type="check" title="Authentication Required" %}
+If you don't see the "Explain with AI" button, make sure that you are logged into the application.
+{% /callout %}
+
+1. **Access the Task**:
+
+   - First, navigate to the Nx Cloud dashboard and locate the task that failed.
+   - Click on the task to open the detailed view.
+
+2. **Initiate AI Explanation**:
+
+   - Within the task details, find the "Explain with AI" button.
+   - Click on this button to start the AI analysis.
+     ![explain with ai button](/nx-cloud/troubleshooting/explain-with-ai-1.png)
+
+3. **Review the Explanation**:
+
+   - The AI will analyze the error log with additional context from the project task and provide a detailed explanation of the failure.
+   - It will also offer suggestions on how to resolve the issue.
+     ![explain response](/nx-cloud/troubleshooting/explain-with-ai-2.png)
+
+4. **Implement the Suggestions**:
+
+   - Go through the AI-generated suggestions carefully.
+   - Apply the recommended changes to your codebase.
+
+5. **Verify the Fix**:
+
+   - After making the changes, rerun the task to see if the issue is resolved.
+
+6. **Mark answer as not helpful** (optional):
+   - If the suggested changes did not help you, click on "Set answer as not helpful". This allows us to continuously improve the responses.
+
+## How to Enable AI Features for Enterprise On-Prem Installations
```

Comment:
this should probs go onto the helm repo instead of here since that seems to be where we are keeping configuration info. or at min should be in both locations. 

https://github.com/nrwl/nx-cloud-helm/

Reply:
It should be in both locations for sure 🙂

Reply:
I think it's fine to have them here 👍 

I've always liked keeping more things in the helm repo though, since there's a lot less friction to making updates: if a change is a made to a chart, any guides or comments can usually be updated in the same PR. And when something is easy, there's a bigger chance it'll get a done 😄 

having them in two places might lead to things being out of sync

in my PR, I changed a bunch of the auth docs to point to Helm directly: https://github.com/nrwl/nx/pull/26862/files#diff-3beb284984421d69a4a5199ed488fd9892d9f9926b381ab4e421c79060f9a528R135

I don't know if that's the best choice, or if it matters that much since we only have a handful of on-prem installations that those docs will apply to, but I think it'll much our job of keeping them up to date easier